diff --git a/node_modules/@docusaurus/theme-classic/lib/theme/Icon/ExternalLink/index.js b/node_modules/@docusaurus/theme-classic/lib/theme/Icon/ExternalLink/index.js
index 672ca9c..78bdc20 100644
--- a/node_modules/@docusaurus/theme-classic/lib/theme/Icon/ExternalLink/index.js
+++ b/node_modules/@docusaurus/theme-classic/lib/theme/Icon/ExternalLink/index.js
@@ -6,7 +6,7 @@
  */
 import React from 'react';
 import styles from './styles.module.css';
-export default function IconExternalLink({width = 13.5, height = 13.5}) {
+export default function IconExternalLink({width = 12, height = 12}) {
   return (
     <svg
       width={width}
@@ -16,7 +16,7 @@ export default function IconExternalLink({width = 13.5, height = 13.5}) {
       className={styles.iconExternalLink}>
       <path
         fill="currentColor"
-        d="M21 13v10h-21v-19h12v2h-10v15h17v-8h2zm3-12h-10.988l4.035 4-6.977 7.07 2.828 2.828 6.977-7.07 4.125 4.172v-11z"
+        d="M24 0v19h-3V5.12L2.5 23.621.379 21.5 18.878 3H5V0z"
       />
     </svg>
   );
diff --git a/node_modules/@docusaurus/theme-classic/src/theme/Icon/ExternalLink/index.tsx b/node_modules/@docusaurus/theme-classic/src/theme/Icon/ExternalLink/index.tsx
index ea618ee..534e3fa 100644
--- a/node_modules/@docusaurus/theme-classic/src/theme/Icon/ExternalLink/index.tsx
+++ b/node_modules/@docusaurus/theme-classic/src/theme/Icon/ExternalLink/index.tsx
@@ -11,8 +11,8 @@ import type {Props} from '@theme/Icon/ExternalLink';
 import styles from './styles.module.css';
 
 export default function IconExternalLink({
-  width = 13.5,
-  height = 13.5,
+  width = 12,
+  height = 12,
 }: Props): JSX.Element {
   return (
     <svg
@@ -23,7 +23,7 @@ export default function IconExternalLink({
       className={styles.iconExternalLink}>
       <path
         fill="currentColor"
-        d="M21 13v10h-21v-19h12v2h-10v15h17v-8h2zm3-12h-10.988l4.035 4-6.977 7.07 2.828 2.828 6.977-7.07 4.125 4.172v-11z"
+        d="M24 0v19h-3V5.12L2.5 23.621.379 21.5 18.878 3H5V0z"
       />
     </svg>
   );
